### PR TITLE
Add configuration for enabling signature validation certificate alias  in the UI.

### PR DESCRIPTION
**Pull request by ChamathNS**
## Purpose
$subject.
Fix https://github.com/wso2/product-is/issues/11015.

### CODE REVIEW

File: en/docs/learn/configuring-saml2-web-single-sign-on.md [hunk 1]
```diff
@@ -123,6 +123,17 @@ To configure SAML2 Web SSO:
     <tr class="even">
     <td><strong>Certificate Alias</strong></td>
     <td><div class="content-wrapper">
+    <p class="admonition-title">Tip</p>
+    <p>From WSO2 IS 5.12.0 onwards, the <strong>Certificate Alias</strong> option would be hidden from the application
```
Comment (gomathyk) [inline en/docs/learn/configuring-saml2-web-single-sign-on.md:127]:
```suggestion
    <p>From WSO2 IS 5.12.0 onwards, the <strong>Certificate Alias</strong> option is hidden from the application
```

File: en/docs/learn/configuring-saml2-web-single-sign-on.md [hunk 2]
```diff
@@ -123,6 +123,17 @@ To configure SAML2 Web SSO:
...
+     configurations. To enable the <strong>Certificate Alias</strong> option in the console UI, add the following
+      configuration to the <code>                &lt;IS_HOME&gt;/repository/deployment.toml</code>.
```
Comment (gomathyk) [inline en/docs/learn/configuring-saml2-web-single-sign-on.md:129]:
```suggestion
      configuration to the <code>                &lt;IS_HOME&gt;/repository/deployment.toml</code> file.
```

File: en/docs/learn/configuring-saml2-web-single-sign-on.md [hunk 3]
```diff
@@ -123,6 +123,17 @@ To configure SAML2 Web SSO:
...
+      
+      ```toml
```
Comment (gomathyk) [inline en/docs/learn/configuring-saml2-web-single-sign-on.md:131]: Please check that the alignment is as follows: 
```toml
[console.applications.ui]
certificate_alias_enabled=true
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/learn/configuring-saml2-web-single-sign-on.md (hunk 3 above)
Comment (ChamathNS) [reply en/docs/learn/configuring-saml2-web-single-sign-on.md]: Addressed with https://github.com/wso2/docs-is/pull/2403/commits/5fead90dcb7a5569141bd5cff0b204d774fa79e6.

**Review by gomathyk — APPROVED** [review]

**Pull request merged**